src/abaqus/Session/SessionBase.py

```diff
@@ -686,4 +687,4 @@ def openOdb(self, name: str, *args, **kwargs) -> Odb:
 
     @staticmethod
     def exit():
-        os.system("exit")
+        sys.exit()
```

Comment:
I think this is the reason. I actually do not what is the difference between this two, but `sys.exit()` just fails.

Reply:
I think that `sys.exit()` is the command to exit the Python interpreter. That is also what says in help:
```python
>>> help(sys.exit)
Help on built-in function exit in module sys:
exit(status=None, /)
    Exit the interpreter by raising SystemExit(status).
```
On the other hand, `os.system('exit')` executes the string command (`exit` in this case) in subshell of the system, and doesn't get out the interpreter in all cases.
```python
>>> import os
>>> os.system('exit')
0
>>> # still in the interpreter 
```
That may be verified if you try to print something on the terminal after the `os.system('exit')`. The Python 3.9 interpreted will print that.

Reply:
Yes, calling `exit` after `openOdb` is a workaround for errors happening when we try to save the data to a file by the local Python interpreter because the data will be saved by the Abaqus/Python interpreter.

Reply:
> I think that `sys.exit()` is the command to exit the Python interpreter. That is also what says in help:
> 
> ```python
> >>> help(sys.exit)
> Help on built-in function exit in module sys:
> exit(status=None, /)
>     Exit the interpreter by raising SystemExit(status).
> ```
> 
> On the other hand, `os.system('exit')` executes the string command (`exit` in this case) in subshell of the system, and doesn't get out the interpreter in all cases.
> 
> ```python
> >>> import os
> >>> os.system('exit')
> 0
> >>> # still in the interpreter 
> ```
> 
> That may be verified if you try to print something on the terminal after the `os.system('exit')`. The Python 3.9 interpreted will print that. I think the intention was to get out Python 3.9 interpreter after the `openOdb` command, is that right?

You mean, if we use `os.system('exit')`, the following codes after `openOdb` will be executed anyway?

Reply:
Then we have to find a workaround to prevent saving data twice in the local Python interpreter and in the Abaqus Python interpreter, maybe write some guide to use this:
```python
data == ...
if data:
    # save the data
```

Reply:
> You mean, if we use os.system('exit'), the following codes after openOdb will be executed anyway?

That's right. I think that will happen.

For instance. I removed the verification `if data.size > 0:` in the `test_odb.py` file.
And then the result `data.csv` gets empty, because Python 3.9 saves an empty data.
```python
...
    if data.size > 0: # removing this, the next line will execute on Python 3.9, after the `openOdb` command
        np.savetxt('data.csv', data, header='time,U3', delimiter=',', comments='')
```

Reply:
> And then the result `data.csv` gets empty, because Python 3.9 saves an empty data.

I am confused because I use this long ago and I did not get empty data before.

Reply:
> Then we have to find a workaround to prevent saving data twice in the local Python interpreter and in the Abaqus 

Yeah, I think that exit the Python 3.9 interpreter after `openOdb` command was a better solution (and I thought that was the intention).

Reply:
> 

That is an excellent solution!

Reply:
> I am confused because I use this long ago and I did not get empty data before.

Yeah, I just made 3 individual tests, all of then seems to get code executed after `os.system('exit')` command.

1. Execute `os.system('exit')` on the Python REPL interpreter, with no script, i.e.:
    ```python
    >>> import os
    >>> os.system('exit')
    0
    >>> # still in the interpreter 
    ```
2. Print messages on the `Session.exit` method, i.e.:
    ```python 
    ...
    @staticmethod
    def exit():
        print('before') # prints before
        os.system('exit')
        print('after') # prints after
    ```
3. Removing the verification of `data.size` in the test, and got an empty file, i.e.
    ```python
    ...
    data = np.array(dataList[0])
    # if data.size > 0: # don't verify
    np.savetxt('data.csv', data, header='time,U3', delimiter=',', comments='')
    ```

Reply:
Yes, you are right, I have ran some tests also.

Reply:
> > Yeah, I think that exit the Python 3.9 interpreter after openOdb command was a better solution (and I thought that was the intention). Maybe there is a way to prevent pytest from fail on this action, I found [this reference](https://stackoverflow.com/questions/30256332/verify-the-error-code-or-message-from-systemexit-in-pytest)
> 
> That is an excellent solution!

I did that now. Updated the PR with 3 commits